src/main/java/org/folio/list/services/ListValidationService.java

```diff
@@ -149,4 +150,10 @@ private void assertUserNotExporting(ListEntity list, ListActions failedAction) {
       throw new ExportInProgressException(list, failedAction);
     }
   }
+  public UUID getCurrentOwnerId(ListEntity entity) {
+    if (entity.getUpdatedBy() == null) {
+      return entity.getCreatedBy();
+    }
+    return entity.getUpdatedBy();
+  }
```

Comment:
Small thing: Looking a little closer at this, I think the `isPrivate` check should be in this method, rather than in `assertSharedOrOwnedByUser()` for the method name to make sense.

But at that point, `assertSharedOrOwnedByUser()` would just be `if (getCurrentOwnerId(.equals(getUserId()) { throw something }`, and `getCurrentOwnerId()`. That's simple enough that I don't think we really gain much by having it split across to methods, so I'd just put it all in `assertSharedOrOwnedByUser()` and remove `getCurrentOwnerId()`

Reply:
@mweaver-ebsco, addressed your comment